pandas/io/parquet.py

```diff
@@ -260,8 +260,13 @@ def read_parquet(path, engine='auto', columns=None, **kwargs):
 
     Parameters
     ----------
-    path : string
-        File path
+    path : various
+        Either a path to a file (a :class:`python:str`,
```

Comment:
can you remove the "python" in the reference? I don't think that it is needed to ensure a correct link) (same for the ones below)